src/fmu/dataio/_definitions.py

```diff
@@ -64,30 +64,13 @@ class ValidFormats:
     )
 
 
-STANDARD_TABLE_INDEX_COLUMNS: Final = {
+STANDARD_TABLE_INDEX_COLUMNS: Final[dict[str, list[str]]] = {
     "inplace_volumes": ["ZONE", "REGION", "FACIES", "LICENCE"],
-    "timeseries": ["DATE"],  # summary
     "rft": ["measured_depth", "well", "time"],
+    "timeseries": ["DATE"],  # summary
     "wellpicks": ["WELL", "HORIZON"],
 }
 
-# This setting will set if subkeys is required or not. If not found in list then
-# assume False.
-CONTENTS_REQUIRED: Final = {
-    "fluid_contact": {"contact": True},
-    "field_outline": {"contact": False},
-    "field_region": {"id": True},
```

Comment:
Minor bug discovered, the fields property and seismic should have been a part of this. (Confirmed with @jcrivenaes )

Reply:
Bug, but will keep it as is for now.